lib/nice_partials/partial.rb

```diff
@@ -29,7 +29,7 @@ def yield(*arguments, &block)
     end
 
     def helpers(&block)
-      class_eval &block
+      self.class.class_eval(&block)
```

Comment:
@seanpdoyle I figured out what was up in #64, we were hitting the `method_missing` hook from `delegate_missing_to :@view_context` and adding the method to `ActionView::Base`, which then also worked on `partial` because of the `method_missing` hook.

If we extend it on our class, we're good.

Now why an instance of `ActionView::Base` responds to `class_eval`, well, I don't understand that.